Sources/_InternalTestSupport/PlatformHelpers.swift

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Basics
+
+public func isMacOs() -> Bool {
```

Comment:
If these are for swift-testing tests it may make more sense to implement these as test traits

Reply:
I think here is Ok; TSC is deprecated and there aren't plans to expand its use.